docs/general/fixtures.md

```diff
@@ -0,0 +1,255 @@
+
+### get-signals-v1-private-signals-id.json
+
+Old Name: incident.json
+
+Endpoint: v1/private/signals/123
+
+Key: results
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/signals/123' "Authorization:Bearer $SIA_TOKEN" | jq '.results' > get-signals-v1-private-signals-id.json
+
+### get-signals-v1-private-categories-id.json
+
+Old name: category.json
+
+Endpoint: v1/private/categories/23
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/categories/23' "Authorization:Bearer $SIA_TOKEN" | jq '.results' > get-signals-v1-private-categories-id.json
+
+### get-signals-v1-private-departments-id.json
+
+Old name: department.json
+
+Endpoint: v1/private/department/6
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/departments/6' "Authorization:Bearer $SIA_TOKEN" | jq > get-signals-v1-private-departments-id.json
+
+### get-signals-v1-private-categories-id-history.json
+
+Old name: history.json
+
+Endpoint: v1/private/categories/23/history.json
+
+Note: There is currently no history on acc
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/categories/23/history' "Authorization:Bearer $SIA_TOKEN" | jq '.results' > get-signals-v1-private-categories-id-history.json
+
+### post-response-signals-v1-private-signals
+
+Old Name: postIncident.json
+
+Source: models/categories/selectors.js
+
+### get-geosearch-bag-lat-on-radius.json
+
+Also used as (with different data): geosearch.json
+
+Old Name: geography.json
+
+Endpoint: geosearch/bag/?lat=52.37188789984033&lon=4.88888741680181&radius=50
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/geosearch/bag/?lat=52.37188789984033&lon=4.88888741680181&radius=50' "Authorization:Bearer $SIA_TOKEN" > get-geosearch-bag-lat-on-radius.json
+
+## Selectors
+
+### selector-models-categories-selectors_makeSelectStructuredCategories.json
+
+Old Name: categories_structured.json
+
+Source: models/categories/selectors.js
+
+Selector: makeSelectStructuredCategories
+
+List keys (categories):
+
+    jq 'keys[]' categories_structured.json
+
+Cheatsheet:
+
+    jq '.results[] | select(.id == 2)' categories_structured.json
+
+### selector-models-categories-selectors_inputRolesSelector.json
+
+Old Name: inputRolesSelector.json
+
+Source: models/categories/selectors.js
+
+Selector: inputRolesSelector
+
+## Other
+
+### shared-services-map-location_formatPDOKResponse.json
+
+Old Name: PDOKResponseData.json
+
+Source: 'shared/services/map-location/index.js'
+
+Method: formatPDOKResponse
+
+Method Data Source:
+    https://geodata.nationaalgeoregister.nl/locatieserver/v3/suggest?fq=gemeentenaam:amsterdam&fl=id,weergavenaam&fq=bron:BAG&fq=type:adres&fl=id,weergavenaam,straatnaam,huis_nlt,postcode,woonplaatsnaam,centroide_ll&q=Kal
+
+#### kto.json
+
+Type: Form data
+
+Origin: unknown
```

Comment:
Why is the origin unknown?

Reply:
Because I don't know where the `fixture data` in `kto.json` data is coming from.